src/3045R5_quantities_and_units_library.md

```diff
@@ -3991,9 +3991,9 @@ of them together with the recommended portable replacements:
 | GREEK SMALL LETTER PI |   π    | u8"\u03c0" |         "pi"         |
 | DOT OPERATOR          |   ⋅    | u8"\u22C5" |      `<none>`*       |
 
-(*) Users should not select `unit_symbol_separator::half_high_dot` and `character_set::portable`
+_NOTE: Users should not select `unit_symbol_separator::half_high_dot` and `character_set::portable`
```

Comment:
This star referred to line 3992, which had this star as well. Do you have a better idea of how to connect those?

Reply:
I didn't know what the star was there for, but this makes sense.  I switched to the same footnote notation I was using elsewhere.  I verified that it works in the rendered output.